src/FormBuilder/resources/js/form-builder/src/settings/offline-donation/index.js

```diff
@@ -5,7 +5,7 @@ import DonationInstructions from './donation-instructions';
 
 const OfflineDonationsSettings = () => {
     const {
-        settings: {offlineDonationsCustomize},
+        settings: {enableOfflineDonations, offlineDonationsCustomize},
```

Comment:
unless i'm missing something,`offlineDonationsCustomize` does not look like it's being used here - if not let's remove.

Reply:
I resolved this with the merge.